test/io.jl

```diff
@@ -11,22 +11,39 @@ extend this in the original package
 
 using GigaSOM, DataFrames, XLSX, CSV
 
+#create genData and data folder and change dir to dataPath
 cwd = pwd()
+if occursin("jenkins", homedir()) || "TRAVIS" in keys(ENV)
+    genDataPath = mktempdir()
+    dataPath = mktempdir()
+    cd(dataPath)
+else
+    if !occursin("test", cwd)
+        cd("test")
+        cwd = pwd()
+    end
+    dataFolders = ["genData", "data"]
+    for i in dataFolders
+        if !isdir(i)
+            mkdir(i)
+        end
+    end
+    genDataPath = cwd*"/genData"
+    dataPath = cwd*"/data"
+    cd(dataPath)
```

Comment:
This line is redundant with line 19 and can be placed outside the `if` condition.

Reply:
done